components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/client/IdentityProviderMgtServiceClient.java

```diff
@@ -299,17 +309,20 @@ public void deleteIdP(String idPName) throws Exception {
      * @throws Exception Error when updating Identity Provider information
      */
     public void updateIdP(String oldIdPName, IdentityProvider identityProvider) throws Exception {
+
         try {
             idPMgtStub.updateIdP(oldIdPName, identityProvider);
         } catch (Exception e) {
             log.error("Error in updating the Identity Provider for a given tenant", e);
-            throw new Exception(e.getMessage());
+            throw new Exception("Error in updating the Identity Provider for a given tenant", e);
```

Comment:
Is it possible to avoid changing the error message?

Reply:
Fixed by reverting.